### ISSUE TITLE
New dev environment bug

### ISSUE THREAD

**Issue by fractal161** [professor-l/classic-tetris-project#23]
When testing with a new development environment in the test bot's Twitch account, the following error is given:
```
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "~/proj/classic-tetris-project/classic_tetris_project/management/commands/bot.py", line 72, in run_twitch
    twitch.client.start()
  File "~/proj/classic-tetris-project/classic_tetris_project/twitch.py", line 78, in start
    self.reactor.process_forever()
  File "~/.virtualenvs/ctp/lib/python3.7/site-packages/irc/client.py", line 842, in process_forever
    consume(infinite_call(one))
  File "~/.virtualenvs/ctp/lib/python3.7/site-packages/more_itertools/recipes.py", line 137, in consume
    deque(iterator, maxlen=0)
  File "~/.virtualenvs/ctp/lib/python3.7/site-packages/irc/client.py", line 823, in process_once
    self.process_data(in_)
  File "~/.virtualenvs/ctp/lib/python3.7/site-packages/irc/client.py", line 787, in process_data
    conn.process_data()
  File "~/.virtualenvs/ctp/lib/python3.7/site-packages/irc/client.py", line 271, in process_data
    self._process_line(line)
  File "~/.virtualenvs/ctp/lib/python3.7/site-packages/irc/client.py", line 304, in _process_line
    handler(arguments, command, source, tags)
  File "~/.virtualenvs/ctp/lib/python3.7/site-packages/irc/client.py", line 340, in _handle_message
    self._handle_event(event)
  File "~/.virtualenvs/ctp/lib/python3.7/site-packages/irc/client.py", line 369, in _handle_event
    self.reactor._handle_event(self, event)
  File "~/.virtualenvs/ctp/lib/python3.7/site-packages/irc/client.py", line 919, in _handle_event
    result = handler.callback(connection, event)
  File "~/proj/classic-tetris-project/classic_tetris_project/twitch.py", line 84, in <lambda>
    self.connection.add_global_handler("pubmsg", lambda c, e: self._handle_message(e, handler))
  File "~/proj/classic-tetris-project/classic_tetris_project/twitch.py", line 97, in _handle_message
    handler(message)
  File "~/proj/classic-tetris-project/classic_tetris_project/management/commands/bot.py", line 70, in on_message
    context.dispatch()
  File "~/proj/classic-tetris-project/classic_tetris_project/commands/command_context.py", line 157, in dispatch
    if not self.dispatch_custom():
  File "~/proj/classic-tetris-project/classic_tetris_project/commands/command_context.py", line 166, in dispatch_custom
    channel = twitch_user.channel
AttributeError: 'NoneType' object has no attribute 'channel'

```

**Comment by professor-l** [comment]
Thanks fractal, it's on to-do.  Much appreciate the help!

**Comment by professor-l** [comment]
This was fixed in PR #29 - closing.
